README.md

```diff
@@ -1,13 +1,16 @@
 # cs-modify
 
-App structure:
+## App structure:
 
 1. app.py
   * The application's home file
 2. /components
   * Contains all logical components
-3. /public
+  * This folder contains:
```

Comment:
Please add in details for the other files that are contained in this folder that are not in the <code>/handlers</code> directory, namely what are these files and what they do.